tests/unit/plugins/test/test_ipaddress.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The test plugin file for netaddr tests
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import unittest  # TestCase, assertTrue
+import ansible_collections.ansible.utils.plugins.test.ipaddress_tests as ipaddress
+from ansible.template import Templar
+
+TESTS = [
```

Comment:
these test won't pass because not all test plugins have been added, once the first is dialed in, the others can be added to this PR and we'll eventually see the tests pass